### PR TITLE
Added Axolotl API

### PR DESCRIPTION
**Pull request by AceKiron**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by adamsofea1111 — APPROVED** [review]

**Review by fintzd — APPROVED** [review]
LGTM I guess

**Comment by AceKiron** [comment]
.... How long do these PRs usually take?

**Comment by fintzd** [comment]
Good question, but I do not know unfortunately!

**Review by adamsofea1111 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ For information on contributing to this project, please see the [contributing gu
 ### Animals
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Axolotl API](https://theaxolotlapi.netlify.app/) | Collection of axolotl pictures and facts | No | Yes | Unkown
```
Comment (marekdano) [inline README.md:63]:
```suggestion
| [Axolotl](https://theaxolotlapi.netlify.app/) | Collection of axolotl pictures and facts | No | Yes | Unknown
```

`API` shouldn't be in the name

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @AceKiron!

Please check my suggestion above.

**Comment by AceKiron** [comment]
Checked and updated it @marekdano

**Comment by AceKiron** [comment]
Wait, now it isn't squashed anymore :/

**Review by adamsofea1111 — APPROVED** [review]

**Review by adamsofea1111 — APPROVED** [review]

**Review by fintzd — APPROVED** [review]
LGTM

**Comment by AceKiron** [comment]
Hey, still waiting for this to hopefully get merged :/

**Comment by matheusfelipeog** [comment]
Hi @AceKiron ✋ 

> Hey, still waiting for this to hopefully get merged :/

Sorry for the delay 😅

Can you [squashed](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all changes into a single commit, please?

**Comment by AceKiron** [comment]
> Can you [squashed](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all changes into a single commit, please?

Sorry, but I just can't get squashing figured out, I'll close this PR and make a new one with a repo that actually has only 1 new commit.

**Comment by AceKiron** [comment]
@matheusfelipeog the new PR is #1886

**Comment by matheusfelipeog** [comment]
Ok @AceKiron, I will review this now ;)

**Pull request closed without merging**